### DISCUSSION TITLE
Access SQLite database remotely by mounting web server via (s)FTP

### DISCUSSION THREAD

**Discussion by philipschilling** [iterate-ch/cyberduck#16255] [Q&A] [answered]
I want to host the file-based database SQLite remotely on a shared web hosting. There exist different local apps to manage the database such as DB Browser for SQLite, SQLiteManager, etc. However, these tools do not have a built-in FTP client as far as I know to understand the database on a remote server.

**Question:**
1. Can I use MountainDuck to mount the remote server via (s)FTP and the use the local database manager apps to edit the database?
2. Can this lead to db inconsistencies when the sqlite database file is accessed from the web server and via the mounted connection at the same time or will the web server take care of both file access connections?

**Comment by AliveDevil** [comment, marked as answer]
Mountain Duck should work fine in this case.

1. Yes, there is no difference in editing a local file or one available through Mountain Duck.
2. Yes, make sure to only edit the file in one active application.
That is: The server may deny Mountain Duck access to the SQLite database, when your web server application holds the database open in a shared-nothing handle. _But_ this is software dependent. Mountain Duck _will not_ create long-living handles to remote files. It will open, fetch and close the file. There is no way to "lock" remote files.
